### DISCUSSION TITLE
How to set absolute values in 'px' ?

### DISCUSSION THREAD

**Discussion by jb-thery** [tailwindlabs/tailwindcss#2077] [Help] [answered]
Hi,

I learn tailwind and I wanna know how to set css property's like "margin-top: 57px;" or "bottom: 20px;" without set new property in tailwind.config.js ?

**Comment by hacknug** [comment, marked as answer]
Use inline-styles or a custom CSS stylesheet I guess? I'm not sure what the actual question here is 🤔

**Reply by adamwathan** [reply]
Yep, either add them to your Tailwind config, or do it in a way that has nothing to do with Tailwind (inline styles, custom CSS).

**Reply by jb-thery** [reply]
Thanks !

**Reply by shihabuddin-dev** [reply]
Need to add this file

**Comment by shihabuddin-dev** [comment]
You need to setup tailwind.config.js
